code/modules/surgery/organs/eyes.dm

```diff
@@ -44,6 +47,29 @@
 	M.update_tint()
 	M.update_sight()
 
+/obj/item/organ/eyes/on_life()
+	..()
```

Comment:
this will cause it to autoheal

Reply:
Definitely will, that was the intent. If an organ is in use and not failing the organ gains an inherited healing factor of .1% of its total health per on_life process, which is every 6-8 seconds. The amount of healing is so minor it's hardly a replacement for oculine. That, and organs do not repair when failing.

Reply:
Organs can have their healing factor changed to be greater or more per on_life process.